### DISCUSSION TITLE
bot-is-typing

### DISCUSSION THREAD

**Discussion by danielehrhardt** [telegraf/telegraf#1792] [Q&A] [answered]
https://stackoverflow.com/questions/61520440/pretending-that-telegram-bot-is-typing

is this possible?

**Comment by danielehrhardt** [comment, marked as answer]
bot.telegram.sendChatAction(ctx.chat.id, "typing");

**Reply by jesus-ale43** [reply]
or
```
ctx.sendChatAction('typing');
```
same thing 😺
